libopenage/log/tail_logsink.h

```diff
@@ -0,0 +1,31 @@
+// Copyright 2015-2015 the openage authors. See copying.md for legal info.
+
+#ifndef OPENAGE_LOG_TAIL_LOGSINK_H_
+#define OPENAGE_LOG_TAIL_LOGSINK_H_
+
+#include "logsink.h"
+
+namespace openage {
+namespace log {
+
+
+class TailSink : public LogSink {
+public:
+	const static unsigned int tail_size = 8;
+	char *tail[tail_size] = {};
```

Comment:
can you use stl-types for that. some queue might be suitable, e.g. http://en.cppreference.com/w/cpp/container/queue